wiki/Front-end_with_curl.md

```diff
@@ -0,0 +1,278 @@
+The parameters key should be <code>backendurl</code> then enter the full HTTP URL you wish to use.
+</p>
+<p>
+An example of a way to add such parameter is below, note that the "?" indicates that parameters has been added to the front-end url.
+<code>Localhost:8080</code> is the front-end url, and we want to access the context path /auth but on URL <code>http://127.0.0.1:8090/</code> that is the back-end we wish to use.
+</p>
+<pre>
+curl -X GET http://localhost:8080/auth?backendurl="http://127.0.0.1:8090/"
+Tomcat Example:
+curl -X GET http://localhost:8080/eifrontend/auth?backendurl="http://127.0.0.1:8090/eibackend/"
+</pre>
+<p>
+This way of entering the backendurl may be the easiest way. It works with GET, POST and PUT requests. Currently entering just a back-end name is not supported.
+</p>
+<p>
+Note: It is not possible to add the backendurl parameter as a JSON parameter.
+</p>
+
+<h3>Endpoints bridged from front-end to back-End</h3>
+<table>
+   <tr>
+      <th>Context Path</th>
+      <th>Type</th>
+      <th>Explenation</th>
+   </tr>
+   <tr>
+      <td>/auth</td>
+      <td>GET</td>
+      <td>Authentication status, is LDAP enabled</td>
+   </tr>
+   <tr>
+      <td>/auth/checkStatus</td>
+      <td>GET</td>
+      <td>Back-end status, is back-end online</td>
+   </tr>
+   <tr>
+      <td>/subscriptions</td>
+      <td>GET</td>
+      <td>Fetches all available subscriptions from the back-end</td>
+   </tr>
+   <tr>
+      <td>/subscriptions</td>
+      <td>POST</td>
+      <td>Request to add a new subscription included as a JSON object</td>
+   </tr>
+   <tr>
+      <td>/subscriptions</td>
+      <td>PUT</td>
+      <td>Request to update a given subscription object included as a JSON object</td>
+   </tr>
+   <tr>
+      <td>/subscriptions/*subscription name*</td>
+      <td>GET</td>
+      <td>Fetches information about a single subscription from the back-end</td>
+   </tr>
+   <tr>
+      <td>/subscriptions/*subscription name*</td>
+      <td>DELETE</td>
+      <td>Deletes a single instance from the back-end</td>
+   </tr>
+   <tr>
+      <td>/information</td>
+      <td>GET</td>
+      <td>Fetches all information from about the back-end and it´s components</td>
+   </tr>
+   <tr>
+      <td>/queryAggregatedObject</td>
+      <td>GET</td>
+      <td>Download a subscription template json file</td>
+   </tr>
+   <tr>
+      <td>/queryMissedNotifications</td>
+      <td>GET</td>
+      <td>Queries any missed notifications</td>
+   </tr>
+   <tr>
+      <td>/query</td>
+      <td>GET</td>
+      <td>Free style query, here the user may specify their queries</td>
+   </tr>
+   <tr>
+      <td>/query</td>
+      <td>POST</td>
+      <td>Free style query, here the user may specify their queries</td>
+   </tr>
+   <tr>
+      <td>/rules/rule-check/testRulePageEnabled</td>
+      <td>GET</td>
+      <td>Gives a result weather the test rules function is enabed in the back-end</td>
```

Comment:
enabled

Reply:
'whether', you might need to add the alternative at the end of the sentence as well, like "in the back-end or not".

Reply:
Done.